src/components/ConversationSettings/ModerationSettings.vue

```diff
@@ -21,6 +21,7 @@
 
 <template>
 	<div>
+		<ListableSettings :token="token" />
```

Comment:
Currently the settings block is captioned with `Meeting settings` which is weird.
Only the lobby refers to meetings. "listable" and "locking" should be `Conversation settings` instead of `Meeting settings`